packages/third-party/tailwind-preset/package.json

```diff
@@ -26,7 +21,15 @@
     "CHANGELOG.md",
     "README.md"
   ],
+  "scripts": {
+    "test": "vitest",
+    "test:coverage": "vitest run --coverage",
+    "test:watch": "vitest watch"
+  },
   "devDependencies": {
-    "tailwindcss": "^3.4.17"
+    "@lynx-js/types": "^3.2.1",
+    "postcss": "^8.4.35",
+    "tailwindcss": "^3.4.17",
+    "vitest": "^1.3.1"
```

Comment:
nitpick: We can remove this and use the `vitest` from workspace root, since there are also `@vitest/coverage-v8` and `@vitest/ui`